rfc-135-updating-routes-in-router.md

```diff
@@ -0,0 +1,53 @@
+# Updating routes in Router
+
+## Summary
+
+This RFC proposes changing the way that routes are updated in Router, moving away from Router API manually updating Router instances and instead allowing Router instances to keep themselves up-to-date via polling MongoDB for changes.
+
+## Problem
+
+Whenever a route needs to be added or updated, Content Store calls into Router API to perform the update. Router API knows the address of every Router instance from a static file that it reads at start-up; it then uses this information to call the `/reload` endpoint on every Route instance to let them know that routes have been updated, so that those instances reload the routes they hold in memory.
```

Comment:
```suggestion
Whenever a route needs to be added or updated, Content Store calls into Router API to perform the update. Router API knows the address of every Router instance from a static file that it reads at start-up; it then uses this information to call the `/reload` endpoint on every Router instance to let them know that routes have been updated, so that those instances reload the routes they hold in memory.
```